doc/manual/command-ref/conf-file.xml

```diff
@@ -36,6 +36,15 @@ client assumes that the daemon has already loaded them.
   <literal>:</literal> will be loaded in reverse order.
 </para>
 
+<para>
```

Comment:
This file should have been deleted, please change `doc/manual/src/command-ref/conf-file-prefix.md` instead.